skate-plugin/src/main/kotlin/com/slack/sgp/intellij/projectgen/DesktopColors.kt

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2022 Slack Technologies, LLC
```

Comment:
```suggestion
 * Copyright (C) 2024 Slack Technologies, LLC
```

Can you fix up the other new files too before you merge?